fedbiomed/node/round.py

```diff
@@ -350,11 +358,14 @@ def run_model_training(
                 )
 
                 results["encrypted"] = True
-                results["encryption_factor"] = self._secure_aggregation.scheme.encrypt(
-                    params=[self._secure_aggregation.scheme.secagg_random],
-                    current_round=self._round,
-                    weight=results['sample_size'],
-                )
+                if self._secure_aggregation.scheme.secagg_random is not None:
+                    results["encryption_factor"] = self._secure_aggregation.scheme.encrypt(
+                        params=[self._secure_aggregation.scheme.secagg_random],
+                        current_round=self._round,
+                        weight=results['sample_size'],
+                    )
+                else:
+                    results["encryption_factor"] = None
```

Comment:
Encryption factor can be `None` by default `results["encryption_factor"]`

```suggestion
                results["encryption_factor"] = None
                if self._secure_aggregation.scheme.secagg_random is not None:
                    results["encryption_factor"] = self._secure_aggregation.scheme.encrypt(
                        params=[self._secure_aggregation.scheme.secagg_random],
                        current_round=self._round,
                        weight=results['sample_size'])                 
```